DSL/com.larsreimann.safeds/src/main/kotlin/com/larsreimann/safeds/emf/Creators.kt

```diff
@@ -1295,6 +1310,48 @@ fun createSdsStarProjection(): SdsStarProjection {
     return factory.createSdsStarProjection()
 }
 
+/**
+ * Returns a new object of class [SdsSchema].
+ */
+fun createSdsSchema(
+    name: String,
+    annotationCalls: List<SdsAnnotationCall> = emptyList(),
+    columns: List<SdsColumn> = emptyList()
+): SdsSchema {
+    return factory.createSdsSchema().apply {
+        this.name = name
+        this.annotationCallList = createSdsAnnotationCallList(annotationCalls)
+        columns.forEach { addColumn(it) }
+    }
+}
+
+/**
+ * Adds a new object of class [SdsSchema] to the receiver.
+ */
+fun SdsCompilationUnit.SdsSchema(
```

Comment:
```suggestion
fun SdsCompilationUnit.sdsSchema(
```

Function names should be lowercase.